org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -640,38 +640,37 @@ private float IntersectQuad(Vector3 start, Vector3 end, float depth)
         }
 
         /// <summary>
-        /// Registers the duple Collider + XRBaseInteractor as triggering proximity.
+        /// Registers the interactable triggering proximity.
         /// </summary>
-        /// <param name="collider">Collider triggering proximity.</param>
-        /// <param name="xrBaseInteractor">Interactor triggering proximity.</param>
-        public void OnProximityEntered(ProximityEnteredEventArgs proximityEnteredEventArgs)
+        /// <param name="xrBaseInteractable">Interactable triggering proximity.</param>
+        public void OnProximityEntered(ProximityHoverEnteredEventArgs proximityEnteredEventArgs)
         {
-            if (proximityEnteredEventArgs.collider != null &&
-                activeCollidersWithInteractor.Add((proximityEnteredEventArgs.collider, proximityEnteredEventArgs.interactor)) &&
-                activeCollidersWithInteractor.Count >= 1)
+            if (activeInteractables.Add(proximityEnteredEventArgs.Interactable as XRBaseInteractable) &&
+                activeInteractables.Count >= 1)
```

Comment:
nit: do we need to check the `Count` if the `Add` was successful? Feels like that would guarantee a count of at least 1

Reply:
Not really.  It is not needed, that's a remnant of then activeInteractables was a List<>.  I have update the code in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/659/commits/d43041173f5cf1176e155c51ff57d1b618a7d29d resolving.